sdk-core/transactions-outcome-parsers/token_management_transactions_outcome_parser.md

```diff
@@ -1 +1,115 @@
-// TBD: will be defined in a future PR.
+## TokenManagementTransactionsOutcomeParser
+
+A class that provides methods for extracting the output of token management transactions.
+i.e. Getting the ticker of a newly issued ESDT token.
+
+Each method should ensure that there is no `signalError` event;
+
+For methods that return multiple things, plain object can be used (JS) or simple DTOs(Py).
```

Comment:
For consistency, how about always return a DTO, even if only one piece of information is extracted? A bit more extensible without having breaking changes. E.g. if we decide that we'd also want to "extract" the minted amount in `parse_issue_fungible`, we only need to add an extra field on the returned DTO.

Reply:
Yes, that makes sense. Changed.